src/components/transmuters/EthTransmuterLooper.tsx

```diff
@@ -0,0 +1,652 @@
+import { useEffect, useMemo, useState } from "react";
+import { Switch } from "../ui/switch";
+import { Button } from "../ui/button";
+import {
+  useAccount,
+  useReadContracts,
+  useSimulateContract,
+  useWaitForTransactionReceipt,
+  useWriteContract,
+} from "wagmi";
+import { formatEther, parseEther } from "viem";
+import { TokenInput } from "../common/input/TokenInput";
+import { formatNumber } from "@/utils/number";
+import { useChain } from "@/hooks/useChain";
+import { isInputZero } from "@/utils/inputNotZero";
+import { useAllowance } from "@/hooks/useAllowance";
+import { toast } from "sonner";
+import { useWriteContractMutationCallback } from "@/hooks/useWriteContractMutationCallback";
+import { EyeOffIcon, EyeIcon } from "lucide-react";
+import { AnimatePresence, m } from "framer-motion";
+import { accordionVariants, accordionTransition } from "@/lib/motion/motion";
+import { tokenizedStrategyAbi } from "@/abi/tokenizedStrategy";
+import { SYNTH_ASSETS, SYNTH_ASSETS_ADDRESSES } from "@/lib/config/synths";
+import { mainnet } from "viem/chains";
+import { TRANSMUTER_LOOPERS_VAULTS } from "@/lib/config/transmuters";
+import {
+  SupportedTransmuterLooperChainId,
+  TransmuterMetadata,
+} from "@/lib/config/metadataTypes";
+import { ScopeKeys } from "@/lib/queries/queriesSchema";
+import { useWatchQuery } from "@/hooks/useWatchQuery";
+
+enum TL_STATIC_STATE_INDICIES {
+  sharesTokenAddress = 0,
+  decimals = 1,
+  shareTokenName = 2,
+  performanceFee = 3,
+  maxDeposit = 4,
+  maxMint = 5,
+  maxRedeem = 6,
+  maxWithdraw = 7,
+  symbol = 8,
+}
+
+enum TL_DYNAMIC_STATE_INDICIES {
+  fullProfitUnlockDate = 0,
+  isShutdown = 1,
+  lastReport = 2,
+  balanceOf = 3,
+  pricePerShare = 4,
+  profitMaxUnlockTime = 5,
+  profitUnlockingRate = 6,
+  totalAssets = 7,
+  totalSupply = 8,
+}
```

Comment:
this is good thinking! 

but, let's make it much more simple:
1. remove these enums completely.
2. see Comment on lines +78 to +174
3. see Comment on lines +178 to +272